releases/latest/kernel/Dockerfile.ubi.adoptopenjdk11

```diff
@@ -3,6 +3,7 @@ ARG LIBERTY_VERSION=20.0.0.1
 ARG LIBERTY_SHA=dfededade132ba37fabbcf07d5004773ee9f3be5
 ARG LIBERTY_BUILD_LABEL=cl200120200108-0300
 ARG LIBERTY_DOWNLOAD_URL=https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/$LIBERTY_VERSION/openliberty-runtime-$LIBERTY_VERSION.zip
+ARG OPENJ9_SCC=true
```

Comment:
hi @ymanton - should this be an `ENV` instead?  I believe we want the default to be true for application images layers that call `configure.sh`, and `ARG`s won't be available in subsequent builds.

Reply:
Sure, I can do that. Alternatively I can make it a negative build arg, e.g. `DISABLE_OPENJ9_SCC` so that we'll build the SCC by default, even if it's unset. Any preference?

Reply:
Ya, both would work.  I tend to lean more towards the `ENV` approach because that would allows us to know, in retrospect, that a running container image was built with OpenJ9 SCC - it may or may not be helpful in serviceability scenarios.

Reply:
Changes made, it's now an env var initialized from a build arg in all of the kernel Dockerfiles. Updated README to document it.